src/main/kotlin/uk/gov/justice/digital/hmpps/assessrisksandneeds/api/controllers/WidgetController.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.justice.digital.hmpps.assessrisksandneeds.api.controllers
+
+import com.fasterxml.jackson.annotation.JsonView
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestMethod
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.RoshRiskWidgetDto
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.api.model.View
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.services.RiskService
+import uk.gov.justice.digital.hmpps.assessrisksandneeds.services.exceptions.ExternalApiEntityNotFoundException
+
+@RestController
+class WidgetController(
```

Comment:
The name Widget sounds UI like, so not sure it has any place in a RestFUL API.

Reply:
Yeah, not particularly happy with the naming of this one either, but didn't want to break the existing endpoint as it has usages. I've created this one to avoid teams repeating themselves and transforming the existing DTO which isn't particularly useful with how we display it in the UI. Happy for suggestions on alternative names here 🤔